src/en/clouds-lxd-advanced.md

```diff
@@ -214,6 +215,59 @@ Now that the cloud and credentials have been added the next step is to create a
 controller. See [Creating a controller][clouds-lxd-creating-a-controller] on
 the main LXD page.
 
+## Charms and LXD profiles
+
+Juju (`v.2.5.0`) has the following functionality for those charms that make use
+of LXD profiles:
+
+ - A validity check is performed on the profile(s) during the deployment of the
```

Comment:
Maybe the skinny on feature generally first? Something like:
"Juju (`v.2.5.0`) allows charms to include a LXD profile. The profile will be applied to a LXD container that the charm is deployed into."

Reply:
Thanks.